pyquil/paulis.py

```diff
@@ -677,6 +692,83 @@ def get_programs(self):
         coefficients = np.array([term.coefficient for term in self.terms])
         return programs, coefficients
 
+    def matrix(self, qubit_mapping={}, n_qubits=None):
+        """Create the matrix representation of self.
+
+        :param n_qubits:  Number of qubits to represent self on. If None, the maximum qubit from
+                         the PauliSum or qubit_mapping is used
+        :param qubit_mapping: A dictionary-like object that maps from :py:class`QubitPlaceholder`
+                              to :py:class:`int`
+        :return: np.matrix representing the PauliTerm
+        """
+
+        # get a list of the qubits in self
+        self_qubits = self.get_qubits()
+        # find maximum real qubit in self for later use
+        max_qubit = 0
+        for qubit in self_qubits:
+            if isinstance(qubit, int):
+                if qubit > max_qubit:
+                    max_qubit = qubit
```

Comment:
We can replace all the lines 706-712 with
```
max_qubit = max([q for q in ps.get_qubits() if isinstance(q, int)])
```

Reply:
Yup, that does look more elegant. Fixed now.